.ci/pytorch/test.sh

```diff
@@ -411,7 +411,9 @@ test_inductor_cpp_wrapper_shard() {
 
   # Run certain inductor unit tests with cpp wrapper. In the end state, we
   # should be able to run all the inductor unit tests with cpp_wrapper.
-  python test/run_test.py --include inductor/test_torchinductor inductor/test_cpu_repro --verbose
+  python test/run_test.py \
+    --include inductor/test_torchinductor inductor/test_max_autotune inductor/test_cpu_repro \
```

Comment:
I didn't add `test_cpu_cpp_wrapper` here because it's already run in `cpp_wrapper` mode (naturally) in the default inductor tests.